know/rdf/parser/viaf_parser.py

```diff
@@ -0,0 +1,29 @@
+# pylint: disable=no-self-use,unused-import,import-error
+import re
```

Comment:
```suggestion
# pylint: disable=no-self-use
```
Nitpick: We'll import `re` later when we need it, no need to do it now. Likewise, why is this import-error not fixed by changing it to `.abstract_parser`? It fixes it in `summarizer.py` doesn't it?

Reply:
There is a need for re as it is being used in the code. Also like I said last night, adding the "." in front of the import creates an actual runtime error but left as is the error is only there because the linter is wrong even though it runs just fine.

Reply:
Oh you're right I missed the use of it later down, didn't realize it was for `EntityType`. I'll look into fixing pylint for import-error cause this really shouldn't be happening

Reply:
ok thanks

Reply:
Did you run the linter on github's side to check that error, because it could just be my end maybe.